.github/workflows/continuous-integration-test.yml

```diff
@@ -0,0 +1,76 @@
+name: build
+on: [push, pull_request, workflow_dispatch]
+
+env:
+  TERM: xterm-256color
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.5.0
+      with:
+        access_token: ${{ github.token }}
+
+    - uses: actions/checkout@v2
+
+    - name: Install Libraries
```

Comment:
Is the library installation not done in `bootstrap`?

Reply:
This is libraries for Ubuntu like ImageMagick and no they're not installed in the bootstrap script.